Documentation~/index.md

```diff
@@ -1,4 +1,8 @@
-We recommend installing the latest version of Unity 2020 LTS via the Unity Hub; [2019.3](https://unity3d.com/get-unity/download) is the minimum.
+# System Requirements
```

Comment:
Aligned with Python for Unity requirements

Reply:
ok, I see where these references to Mac come from. The Python package is supported for Mac but (I think) not the SG package